src/core_modules/capture-core/src/components/D2Form/field/buildField.js

```diff
@@ -201,6 +201,7 @@ const fieldForTypes = {
     [elementTypes.TIME]: (metaData: MetaDataElement) => getBaseTextField(metaData),
     [elementTypes.PERCENTAGE]: (metaData: MetaDataElement) => getBaseTextField(metaData),
     [elementTypes.URL]: (metaData: MetaDataElement) => getBaseTextField(metaData),
+    [elementTypes.ORG_UNIT_TREE]: (metaData: MetaDataElement) => getBaseTextField(metaData),
```

Comment:
Shoud this insert the org unit field component? it seems to include a basic text field?

Reply:
I had no idea how to include the element type except copy code and typing for _elementTypes_